### DISCUSSION TITLE
[BUG]

### DISCUSSION THREAD

**Discussion by 137231716** [Chia-Network/chia-blockchain#7759] [Support]
My wallet is upgrade from 0.02 to 0.09, found that a lot has changed:
My address is 0.02 purse: txch1vheajn494rlpcdld7g52zvpqhvqf07yatytzq9n69p77avrw985sr8s3kk
Height: 133516
There are 418 TXCH.
Now, address into tsit1vheajn494rlpcdld7g52zvpqhvqf07yatytzq9n69p77avrw985sh3zpsp 0.09 wallet
, tokens disappeared.

**Comment by keliew** [comment]
TXHC is Chia's testnet.

TSIT is...not Chia.

**Comment by 137231716** [comment]
I this how long mining, is the air?
On July 20, why it can also run to?






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Chia-Network/chia-blockchain"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年7月23日(星期五) 下午2:57
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Chia-Network/chia-blockchain] [BUG] (#7742)





 
TXHC is Chia's testnet.
 
TSIT is...not Chia.
